app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt

```diff
@@ -77,6 +79,15 @@ class PinPasswordActivityTest {
       .inject(this)
   }
 
+  @Test
+  fun testPinPasswordActivityWithAdmin_activityVisible_checkKeyboardVisibilityByDefault() {
+    ActivityScenario.launch<PinPasswordActivity>(PinPasswordActivity.createPinPasswordActivityIntent(context, adminPin, userId)).use {
```

Comment:
pass `adminId` instead of `userId`

Reply:
Thank you @rt4914 for suggesting, I have made above changes in my latest commit.